op-node/node/config.go

```diff
@@ -17,6 +17,8 @@ type Config struct {
 	L2     L2EndpointSetup
 	L2Sync L2SyncEndpointSetup
 
+	Coordinator CoordinatorConfig
```

Comment:
we could merge the `CoordinatorConfig` into `driver.Config` to treat it as part config of sequencer? 
this could make code simpler i think

Reply:
Make sense to me. I will update soon.

Reply:
Updated at https://github.com/bnb-chain/opbnb/pull/13/commits/17113eba408dd7f7208f9eac785bfc2871de6a1e. PTAL.